server/.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+build
+
```

Comment:
Here is an example of the js, node.js gitignore for your reference.
https://github.com/github/gitignore/blob/main/Node.gitignore

Reply:
I will added this one you share, thanks :)